eng/pipelines/templates/jobs/archetype-sdk-integration.yml

```diff
@@ -127,7 +133,17 @@ jobs:
             TestType: "node"
             DependencyVersion: same
             NodeTestVersion: "12.x"
-            TestResultsFiles: "**/test-results.xml"            
+            TestResultsFiles: "**/test-results.xml"   
+
+        # Add matrix entry for canary testing
+        ${{ if eq(parameters.TestCanary, 'true') }}:
+          CanaryTest_Node:
+            OSVmImage: "ubuntu-18.04"
+            TestType: "node"
+            NodeTestVersion: "12.x"
+            TestResultsFiles: "**/test-results.xml"
+            SubscriptionConfiguration: $(sub-config-azure-cloud-test-resources-preview)
+            ResourceGroupLocation: ${{ parameters.ResourceGroupLocationCanary }}
```

Comment:
Do we need a different parameter for this? Would you need this to be different then the default ResourceGroupLocation parameter?

Reply:
Yes, @danieljurek mentioned that the "Central US EUAP" region is usually meant for the canary deployments.
When asked, the event-hubs service team suggested 'Central US EUAP' and 'EAST US 2 EUAP' to be the canary regions.